packages/xo-web/src/xo-app/home/sr-item.js

```diff
@@ -18,6 +18,14 @@ import styles from './index.css'
 
 @connectStore({
   container: createGetObject((_, props) => props.item.$container),
+  isHa: createSelector(
+    (_, props) => props.item,
+    (state, props) => {
+      const _createGetObject = cb => createGetObject(cb)(state, props)
+      return _createGetObject((_, props) => props.item.$poolId).haStatefiles.map(id => _createGetObject(() => id))
+    },
```

Comment:
This selector will be called very often, maybe it should be a selector depending on the pool?

Also, this selector is on each SR item, maybe we should do it at the parent level and construct a single set of HA SR?